### ISSUE TITLE
Failure to reproduce the reported FTAI candidate recall on wikikg90m-v2 dataset

### ISSUE THREAD

**Issue by anyezhiya368** [alipay/Parameter_Inference_Efficient_PIE#3]
Hi! I am reproducing the candidate generation result on the WikiKG90Mv2 dataset but I cannot get the reported result with the provided code and hyperparameters. Is this normal and is there any way to fix this problem? Many thanks!

**Comment by wwwjs** [comment]
Hello, I haven't got the results in the report. What is the recall of your candidate set in the valid set

**Comment by chocolate9624** [comment]
The recall@20k on valid set is close to 67%.

**Comment by wwwjs** [comment]
Are the hyperparameters the default of the current code? My recall@20k on the valid set is only 62%. How many steps have you trained to achieve this performance.

**Comment by chocolate9624** [comment]
The hyperparameters should be the same to the current code.  The checkpoint is chosen based on the node classfication performance on validation set rather than the recall@20k for WikiKG90Mv2 setting .

**Comment by wwwjs** [comment]
Yes, the checkpoint is selected based on the node classification performance. The valid-MRR of my best node classification is 0.69, but the recall@20k of candidate generated is only 0.62.

**Comment by coddinglxf** [comment]
I have a similar problem, I used the default parameters and I can only get about 63% recall@20k, what do you think about this, thanks
